src/query/exceptions.hpp

```diff
@@ -118,11 +118,7 @@ class QueryRuntimeException : public QueryException {
 class HintedAbortError : public utils::BasicException {
  public:
   using utils::BasicException::BasicException;
-  HintedAbortError()
-      : utils::BasicException(
-            "Transaction was asked to abort, most likely because it was "
-            "executing longer than time specified by "
-            "--query-execution-timeout-sec flag.") {}
+  HintedAbortError() : utils::BasicException("Transaction was asked to abort") {}
```

Comment:
You can extend messsage to say: "... either was executing longer than time specified on ... or someone else asked to abort"